src/frontend/Sider/Behandling/Inngangsvilkår/RegisterAktivteter.tsx

```diff
@@ -2,60 +2,80 @@ import React from 'react';
+            </Alert>
+        );
+    }
+
     return (
-        <div>
+        <VStack>
             <Heading size="small">Brukers registrerte aktiviteter</Heading>
-            <Tabell size={'small'}>
-                <Table.Header>
-                    <Table.Row>
-                        <Table.HeaderCell scope="col">Type</Table.HeaderCell>
-                        <Table.HeaderCell scope="col">Status</Table.HeaderCell>
-                        <Table.HeaderCell scope="col">Startdato</Table.HeaderCell>
-                        <Table.HeaderCell scope="col">Sluttdato</Table.HeaderCell>
-                        <Table.HeaderCell scope="col">Aktivitetsdager</Table.HeaderCell>
-                        <Table.HeaderCell scope="col"></Table.HeaderCell>
-                    </Table.Row>
-                </Table.Header>
-                <Table.Body>
-                    {aktiviteter.map((aktivitet) => {
-                        return (
-                            <Table.Row key={aktivitet.id}>
-                                <Table.DataCell>{aktivitet.type}</Table.DataCell>
-                                <Table.DataCell>{aktivitet.typeNavn}</Table.DataCell>
-                                <Table.DataCell>
-                                    {formaterNullableIsoDato(aktivitet.fom)}
-                                </Table.DataCell>
-                                <Table.DataCell>
-                                    {formaterNullableIsoDato(aktivitet.tom)}
-                                </Table.DataCell>
-                                <Table.DataCell>{aktivitet.antallDagerPerUke}</Table.DataCell>
-                                <Table.DataCell>
-                                    {/*Skjules fordi knappen ikke har noen funksjon enda*/}
-                                    {false && (
+            {aktivitetGrunnlag.aktiviteter.length > 0 ? (
+                <Tabell size={'small'}>
+                    <Table.Header>
+                        <Table.Row>
+                            <Table.HeaderCell scope="col">Type</Table.HeaderCell>
+                            <Table.HeaderCell scope="col">Status</Table.HeaderCell>
+                            <Table.HeaderCell scope="col">Startdato</Table.HeaderCell>
+                            <Table.HeaderCell scope="col">Sluttdato</Table.HeaderCell>
+                            <Table.HeaderCell scope="col">Aktivitetsdager</Table.HeaderCell>
+                            <Table.HeaderCell scope="col"></Table.HeaderCell>
+                        </Table.Row>
+                    </Table.Header>
+                    <Table.Body>
+                        {aktivitetGrunnlag.aktiviteter.map((aktivitet) => {
+                            return (
+                                <Table.Row key={aktivitet.id}>
+                                    <Table.DataCell>{aktivitet.type}</Table.DataCell>
+                                    <Table.DataCell>{aktivitet.typeNavn}</Table.DataCell>
```

Comment:
Type vises i kolonnen for status (fra før)
```suggestion
                                    <Table.DataCell>{aktivitet.typeNavn}</Table.DataCell>
                                    <Table.DataCell>{tekstEllerKode(statutAktivitetTilTekst, aktivitet.status)}</Table.DataCell>
```

Reply:
Fikset.